README.md

```diff
@@ -43,8 +43,8 @@ Maven dependencies:
 Gradle dependencies: 
 
 ```
-compile("com.newrelic.telemetry:telemetry:0.6.1")
-compile("com.newrelic.telemetry:telemetry-http-okhttp:0.6.1")
+api("com.newrelic.telemetry:telemetry:0.6.1")
+implementation("com.newrelic.telemetry:telemetry-http-okhttp:0.6.1")
```

Comment:
this change feels weird to me, since we don't know if the user of this library will expose our APIs in theirs or not. But, I also know that `compile` is deprecated, so I honestly have no idea what we should put here!  :)

Reply:
Maybe @jeffalder has an opinion on this?

Reply:
I would, by default, recommend `implementation`. If they want to expose our classes from one project to another, the compiler will complain, and then they can switch their own dependency to `api`. When I managed the change for the agent, I changed _only_ the dependencies that required exposure. There were only 11 `api` usages versus 1045 instances of `implementation`.

Reply:
I'm happy with that.